AENS.md

```diff
@@ -347,6 +381,32 @@ the name enters the `revoked` state. After a fixed timeout of
 2016 blocks, the name will be available for claiming again.
 
 
+### Subname
+
+```
+ ------------ ----
+| name       | 63 |
+ ------------ ----
+| definition |    |
+ ------------ ----
+```
+
+The `subname` transaction MUST be signed by the owner of the `name` entry.
+
+`subname` transaction is used to define a tree of subnames below `name` entry
```

Comment:
Ambiguous. A "tree" may be understood as implying that each subname has a parent from label point of view.